rainier-core/src/main/scala/com/stripe/rainier/core/SBC.scala

```diff
@@ -15,10 +15,15 @@ final case class SBC[L, T](priorGenerators: Seq[Generator[Double]],
 
   val priorGenerator = Generator.traverse(priorGenerators)
 
+  def posteriorSamples(nSamples: Int)(implicit rng: RNG) =
+    posterior.map { case (l, _) => ev(l).generator }.sample(nSamples)
+
   def animate(sampler: Sampler,
               warmupIterations: Int,
               syntheticSamples: Int,
-              logBins: Int = 3)(implicit rng: RNG): Unit = {
+              logBins: Int = 3,
+              nSamples: Int = 10)(implicit rng: RNG): Unit = {
+    val newGoldset = posteriorSamples(nSamples)
```

Comment:
since we have the `Real` version of the posterior (which we use for binning), should we use that also for the goldset? Then our goldset is always a `List[Double]` which simplifies things.

Reply:
👍 I switched to use `RandomVariable.record` on the real part of the posterior to get a `List[Double]`.